examples/nlp_seq/models.py

```diff
@@ -90,146 +69,150 @@ def init(key, shape, dtype=np.float32):
 
   return init
 
-
 class AddPositionEmbs(nn.Module):
-  """Adds learned positional embeddings to the inputs."""
+  """Adds (optionally learned) positional embeddings to the inputs.
 
-  def apply(self,
-            inputs,
-            max_len=2048,
-            posemb_init=nn.initializers.normal(stddev=1.0)):
+  Args:
+    config: TransformerConfig dataclass containing hyperparameters.
+  """
+  config: TransformerConfig
+
+  @nn.compact
+  def __call__(self,
+               inputs,
+               inputs_positions=None):
```

Comment:
(Here and elsewhere)

Can we make all arguments to `__call__()` kw-only?
Ideally, we would also add a type annotation.

See also https://github.com/google/flax/issues/231#issuecomment-716568710

Reply:
done. left type annotations out (for now?)